src/app_service/service/community/dto/community.nim

```diff
@@ -239,6 +239,21 @@ proc parseCuratedCommunities*(response: RpcResponse[JsonNode]): seq[CuratedCommu
         available: false,
         communityId: communityId.getStr()
       ))
+
+proc parseCuratedCommunities*(response: JsonNode): seq[CuratedCommunity] =
+  if (response["communities"].kind == JObject):
+    for (communityId, communityJson) in response["communities"].pairs():
+      result.add(CuratedCommunity(
+        available: true,
+        communityId: communityId,
+        community: communityJson.toCommunityDto()
+      ))
+  if (response["unknownCommunities"].kind == JArray):
+    for communityId in response["unknownCommunities"].items():
+      result.add(CuratedCOmmunity(
```

Comment:
That's a typo here: `CuratedCOmmunity`

Reply:
I'm actually curious to see if the CI is green in this run ;)

Reply:
Wow, I don't get how this was even compiling.
Updated this.